qa/TL3_EfficientNet_benchmark/test_pytorch.sh

```diff
@@ -79,6 +80,16 @@ python multiproc.py --nproc_per_node 8 ./main.py --amp --static-loss-scale 128 -
 # Optimized PyTorch with AutoAugment:
 python multiproc.py --nproc_per_node 8 ./main.py --amp --static-loss-scale 128 --batch-size 512 --workers 10 --epochs 3 --no-checkpoints --training-only --data-backend pytorch_optimized --automatic-augmentation autoaugment --workspace $RESULT_WORKSPACE --report-file bench_report_optimized_pytorch_aa.json $PATH_TO_IMAGENET
 
+# DALI proxy without automatic augmentations
+python multiproc.py --nproc_per_node 8 ./main.py --amp --static-loss-scale 128 --batch-size 512 --workers 13 --epochs 3 --no-checkpoints --training-only --data-backend dali_proxy --automatic-augmentation disabled  --workspace $RESULT_WORKSPACE --report-file bench_report_dali_proxy.json $PATH_TO_IMAGENET
+
+# DALI proxy with AutoAugment
+python multiproc.py --nproc_per_node 8 ./main.py --amp --static-loss-scale 128 --batch-size 128 --workers 13 --epochs 3 --no-checkpoints --training-only --data-backend dali_proxy --automatic-augmentation autoaugment  --workspace $RESULT_WORKSPACE --report-file bench_report_dali_proxy_aa.json $PATH_TO_IMAGENET
+
+# DALI proxy with TrivialAugment
+python multiproc.py --nproc_per_node 8 ./main.py --amp --static-loss-scale 128 --batch-size 128 --workers 13 --epochs 3 --no-checkpoints --training-only --data-backend dali_proxy --automatic-augmentation trivialaugment --workspace $RESULT_WORKSPACE --report-file bench_report_dali_proxy_ta.json $PATH_TO_IMAGENET
+
+# -----
```

Comment:
Probably this is just my preference, can you move it to L70 so we have all cases with DALI grouped together?